quarkc/command.py

```diff
@@ -16,9 +16,9 @@
 Quark compiler.
 
 Usage:
-  quark [options] install [ --java | --python | --javascript | --all ] <file>...
-  quark [options] compile [ -o DIR ] [ --java | --python | --javascript | --all ] <file>...
-  quark [options] run ( --java | --python | --javascript ) <file>...
+  quark [options] install [ (--java | --python | --javascript | --ruby)... | --all ] <file>...
+  quark [options] compile [ -o DIR ] [ (--java | --python | --javascript | --ruby)... | --all ] <file>...
```

Comment:
@keleshev Since this was done in a merge I do not look forward re-doing I vote for a patch on top